src/wp-admin/includes/post.php

```diff
@@ -1961,15 +1961,75 @@ function wp_create_post_autosave( $post_data ) {
 		 * @param array $new_autosave Post array - the autosave that is about to be saved.
 		 */
 		do_action( 'wp_creating_autosave', $new_autosave );
-
+		wp_autosave_post_revisioned_meta_fields( $new_autosave );
```

Comment:
Wonder if we could hook `wp_autosave_post_revisioned_meta_fields` into like this. 

```
add_action( 'wp_creating_autosave', 'wp_autosave_post_revisioned_meta_fields' );
```

To make this unhookable...

Reply:
Great idea! looks like this would work in both places that we call `wp_autosave_post_revisioned_meta_fields` directly.

Reply:
Added in [`750dcdc` (#4859)](https://github.com/WordPress/wordpress-develop/pull/4859/commits/750dcdc81cbcc1de7085d776516e87fef9c70938)